Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
```

Comment:
It was suggested in a previous PullRequest that this might be replaced with an Enum, selecting either rectangles or image rendering. I am not wholly against this, but rectangles do not lend themselves to interpolated rendering, so I would be disinclined to give them the same prestige as the image rendering.